tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/SdkBuildTasks/Utilities/NetSdkTaskLogger.cs

```diff
@@ -87,6 +87,7 @@ public void LogException(Exception ex)
             else
             {
                 Debug.WriteLine(ex.ToString());
+                throw ex;
```

Comment:
As an FYI when you throw an exception in this manor you looks the original callstack because it gets overwritten with the new callstack of where it is getting thrown here.

Reply:
thank you. Will make the change to preserve the original call stack.

Reply:
I am marking this as resolved as if when you can suggest me an alternative, I will be more than happy to fix that code path.

Reply:
There isn't any great way to do this while passing the exception in. If you want to preserve the original callstack you need to do the `throw` directly in the catch block and not in this method. This isn't a huge deal I just wanted to let you know about it.